operator/standalone/delete_test.go

```diff
@@ -0,0 +1,31 @@
+package standalone
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestDeleteStandalonePipeline_IsHelmReleaseDeleted(t *testing.T) {
+	tests := map[string]struct {
+		givenValue    bool
+		expectedValue bool
+	}{
+		"GivenDeleteStandalonePipeline_WhenReleaseNotDeleted_ThenHelmReleaseDeletedFalse": {
+			givenValue:    false,
+			expectedValue: false,
+		},
+		"GivenDeleteStandalonePipeline_WhenReleaseDeleted_ThenHelmReleaseDeletedTrue": {
+			givenValue:    true,
+			expectedValue: true,
+		},
+	}
+	for name, tc := range tests {
+		t.Run(name, func(t *testing.T) {
+			d := &DeleteStandalonePipeline{
+				helmReleaseDeleted: tc.givenValue,
+			}
+			actualIsDeleted := d.isHelmReleaseDeleted(nil)
+			assert.Equal(t, tc.expectedValue, actualIsDeleted)
```

Comment:
That function under test really is just a getter that returns a single boolean value. I don't see a need to test this.

Reply:
It's ok to have it, in case the function will get more business logic in the future, do you agree?

Reply:
That was my initial thought, when I saw it. I agree.

Reply:
If the function gets more logic, hopefully the author and reviewer ensure that the test code is added/changed accordingly as well.
But now, it seems over-engineered, we have 20+ LoC to maintain and understand the test code for 1 loc of business code... I'd prefer having to read less code and not do too much engineering for "eventualities".